sktime/forecasting/base/adapters/_pmdarima.py

```diff
@@ -117,45 +131,162 @@ def _predict_in_sample(
             return y_pred
 
     def _predict_fixed_cutoff(
         self, fh, X=None, return_pred_int=False, alpha=DEFAULT_ALPHA
     ):
-        # make prediction
+        """Make predictions out of sample.
+
+        Parameters
+        ----------
+        fh : array-like
+            The forecasters horizon with the steps ahead to to predict.
+            Default is
+            one-step ahead forecast, i.e. np.array([1]).
+
+        Returns
+        -------
+        y_pred : pandas.Series
+        Returns series of predicted values.
+        """
         n_periods = int(fh.to_relative(self.cutoff)[-1])
         result = self._forecaster.predict(
             n_periods=n_periods,
             X=X,
-            return_conf_int=return_pred_int,
-            alpha=alpha,
+            return_conf_int=False,
+            alpha=DEFAULT_ALPHA,
         )
 
         fh_abs = fh.to_absolute(self.cutoff)
         fh_idx = fh.to_indexer(self.cutoff)
         if return_pred_int:
-            y_pred, pred_int = result
-            y_pred = pd.Series(y_pred[fh_idx], index=fh_abs)
-            pred_int = pd.DataFrame(
-                pred_int[fh_idx, :], index=fh_abs, columns=["lower", "upper"]
-            )
-            return y_pred, pred_int
+            pred_ints = []
+            for a in alpha:
+                result = self._forecaster.predict(
+                    n_periods=n_periods,
+                    X=X,
+                    return_conf_int=True,
+                    alpha=a,
+                )
+                pred_int = result[1]
+                pred_int = pd.DataFrame(
+                    pred_int[fh_idx, :], index=fh_abs, columns=["lower", "upper"]
+                )
+                pred_ints.append(pred_int)
+            return result[0], pred_ints
         else:
             return pd.Series(result[fh_idx], index=fh_abs)
 
+    def _predict_quantiles(self, fh, X=None, alpha=DEFAULT_ALPHA):
+        """
+        Compute/return prediction quantiles for a forecast.
+
+        Must be run *after* the forecaster has been fitted.
+
+        If alpha is iterable, multiple quantiles will be calculated.
+
+        Parameters
+        ----------
+        fh : int, list, np.array or ForecastingHorizon
+            Forecasting horizon, default = y.index (in-sample forecast)
+        X : pd.DataFrame, optional (default=None)
+            Exogenous time series
+        alpha : float or list of float, optional (default=[0.05, 0.95])
+            A probability or list of, at which quantile forecasts are computed.
+
+        Returns
+        -------
+        quantiles : pd.DataFrame
+            Column has multi-index: first level is variable name from y in fit,
+                second level being the values of alpha passed to the function.
+            Row index is fh. Entries are quantile forecasts, for var in col index,
+                at quantile probability in second col index, for the row index.
+        """
+        self.check_is_fitted()
```

Comment:
this line is unnecessary, already checked in the public method

Reply:
Removed